src/main/java/seedu/address/MainApp.java

```diff
@@ -74,7 +74,7 @@ public void init() throws Exception {
 
         logic = new LogicManager(model);
 
-        security = new SecurityManager(true, model, logic);
+        security = new SecurityManager(false, model, logic);
```

Comment:
What's this boolean for?

Reply:
Its to enable or disable to security feature to make it easier for you all to test your features without having to log in all the time. It will be removed before we release